include/spirv2clc.h

```diff
@@ -21,15 +21,19 @@
 #include <vector>
 
 #include "opt/ir_context.h"
+#include <libspirv2clc_export.h>
```

Comment:
Technically, the header should be included with double quotes rather than angle brackets (since we don't look for it in the system search path).